source/VehicleCommander/mapconfig.xml

```diff
@@ -39,21 +39,21 @@
         <!--layer name="Bihsud Bridge and Environs" visible="true" type="LocalDynamicMapLayer" basemap="false">
             <datasetpath>../../BihsudBridgeVCP.mpk</datasetpath>
         </layer> -->
-        <layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
+        <!--layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
```

Comment:
This COA file was a nice way to test (especially control measures) - funnily enough the version installed in the Qt SDK is over San Francisco ArcGIS SDKs\Qt10.X\sdk\samples\data\symbology if you want to use that (I can provide if you don't have Qt SDK installed - I created that version by just moving the previous VC message file to SanFran)